src/config.rs

```diff
@@ -91,7 +90,7 @@ pub fn toggle_autostart() {
                 format!("{}\n\nauto_start: true", config_str)
             };
 
-            let _ = fs::write(&config_path, updated_config);
+            write(&config_path, updated_config).expect("Can not write to config.");
```

Comment:
I have never worked with `anyhow`, so I don't know what it would do in this case. I would be in favor of using `Result` but like you would suggest to do this in a separate PR.